styles/variables.scss

```
@@ -1,4 +1,58 @@
 // Color Variables
-$primary-color: #ff9900;
-$secondary-color: #20008d;
-$ternary-color: #dddddd;
+
+//Primary Colors
+$primary-color-50: #ff9900;
+$primary-color-100: #ffdfb0;
+$primary-color-200: #ffd08a;
+$primary-color-300: #ffbb54;
+$primary-color-400: #ffad33;
+$primary-color-500: #ff9900;
+$primary-color-600: #e88b00;
+$primary-color-700: #b56d00;
+$primary-color-800: #8c5400;
+$primary-color-900: #6b4000;
+
+// Secondary Colors
+
+$neutral-color-50: #eaeaea;
+$neutral-color-100: #bebebe;
+$neutral-color-200: #9f9f9f;
+$neutral-color-300: #737373;
+$neutral-color-400: #585858;
+$neutral-color-500: #2e2e2e;
+$neutral-color-600: #2a2a2a;
+$neutral-color-700: #212121;
+$neutral-color-800: #191919;
+$neutral-color-900: #131313;
+
+// Shade Colors
+
+$shade-white: #ffffff;
+$shade-black: #000000;
+
+//Font Sizes Variables
+$font-title: 60px;
+$font-h1: 48.83px;
+$font-h2: 39.06px;
+$font-h3: 31.25px;
+$font-h4: 25px;
+$font-h5: 20px;
+$font-body-large: 16px;
+$font-body-small: 14px;
+$font-caption-large: 12px;
+$font-caption-small: 10px;
+
+$font-weights: (
+  "regular": 400,
+  "medium": 500,
+  "semi-bold": 600,
+  "bold": 700,
+);
+
+//Logo sizes
+$logoSizes: (
```

lets standarize the hypenated (with the **-**) words for sass variables **"$some-name"**
```suggestion
$logo-sizes: (
```